soc/arm/qemu_cortex_a53/mmu_regions.c

```diff
@@ -11,20 +11,23 @@
 
 static const struct arm_mmu_region mmu_regions[] = {
 
-	MMU_REGION_FLAT_ENTRY("GIC",
-			      DT_REG_ADDR_BY_IDX(DT_INST(0, arm_gic), 0),
-			      DT_REG_SIZE_BY_IDX(DT_INST(0, arm_gic), 0),
-			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_DEFAULT_SECURE_STATE),
+	MMU_REGION_FLAT_ENTRY(
+		"GIC",
+		DT_REG_ADDR_BY_IDX(DT_INST(0, arm_gic), 0),
+		DT_REG_SIZE_BY_IDX(DT_INST(0, arm_gic), 0),
+		MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_DEFAULT_SECURE_STATE | MT_OVERWRITE),
```

Comment:
I was actually wrong suggesting this change. The `MT_OVERWRITE` should not be necessary for the MMIO regions because they are not overlapping with anything (being MMIO).